script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -1126,12 +1137,14 @@ Well, if push comes to shove, I can always take shelter at your house."
 -- Page 147, Offset 842.
 -- 「なーんですかそれー。ざっけんな、オレん家はおまえのビバークポイントじゃねえですよー。休みになりゃあ泊まりにきやがって、おかげで姉貴はオレより友達の遠野くんびいきになっちまって、年々小遣い減ってるんですけどぉー！」
 // This is an elongated "what", not a stutter.
-"Whaaaaat the hell? Screw you, my house isn't your bivouac spot. During days off, you'd come over to stay overnight. Thanks to that, my sister has become more partial to you, my friend Tohno-kun, than me. Now my allowance is decreasing every year!"
+"Whaaaaat? Screw you, my house isn't your bivouac spot.
+ Because you kept staying over on your days off, my sister has ended up becoming more partial to my friend, Tohno-kun, than she is to me.
+ She's started %{i}decreasing%{/i} my allowance every year!"
```

Comment:
That second sentence is still a little strange and lacking in character. Did need changing though.

>Thanks to you staying over damn every day you got off, my sister's gotten more attached to %{i}my%{/i} friend than she is to me. My allowance has been dropping every year since!